tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/jvm/XMLTest.java

```diff
@@ -174,17 +174,19 @@ public void testObjectLevelXML() {
                 "<p:person xmlns:p=\"foo\" xmlns:q=\"bar\">hello</p:person>");
     }
 
-    @Test(enabled =  false) // todo: re-enable when implementing getElement syntax (seq.<elm>)
+    @Test
     public void testFieldBasedAccess() {
         BValue[] returns = BRunUtil.invoke(compileResult, "testFieldBasedAccess");
         Assert.assertEquals(returns[0].stringValue(),
                 "<fname><foo>1</foo><bar>2</bar></fname><lname1><foo>3</foo><bar>4</bar></lname1><fname><foo>5</foo>" +
                         "<bar>6</bar></fname><lname2><foo>7</foo><bar>8</bar></lname2>apple");
         Assert.assertEquals(returns[1].stringValue(), "<fname><foo>1</foo><bar>2</bar></fname>");
-        Assert.assertEquals(returns[2].stringValue(), "<foo>5</foo>");
-        Assert.assertEquals(returns[3].stringValue(), "<foo>5</foo>");
-        Assert.assertEquals(returns[4].stringValue(), "<bar>4</bar>");
-        Assert.assertEquals(returns[5].stringValue(),
+        Assert.assertEquals(returns[2].stringValue(), "<foo>1</foo><foo>5</foo>");
```

Comment:
@rdhananjaya, can you please verify the fixes for this XML test?

Reply:
changes and the results looks good, shall we also change the function name.

Reply:
Thanks, fixed in af0de38.